linera-storage/src/db_storage.rs

```diff
@@ -207,6 +207,28 @@ pub static WRITE_EVENT_COUNTER: LazyLock<IntCounterVec> = LazyLock::new(|| {
     )
 });
 
+/// The metric counting how often the network description is read from storage.
+#[cfg(with_metrics)]
+#[doc(hidden)]
+pub static READ_NETWORK_DESCRIPTION: LazyLock<IntCounterVec> = LazyLock::new(|| {
+    register_int_counter_vec(
+        "network_description",
+        "The metric counting how often the network description is read from storage",
+        &[],
+    )
+});
+
+/// The metric counting how often the network description is written to storage.
```

Comment:
(This one should really be always 1, should it? :sweat_smile:)

Reply:
That's right. We could also refuse to overwrite the network description btw

Reply:
This may change in the future when initialization becomes idempotent after @bart-linera's work.

Reply:
Turns out the dual store does call initialize several times